comp/core/workloadmeta/collectors/internal/kubeapiserver/kubeapiserver_test.go

```diff
@@ -82,14 +82,28 @@ func TestStoreGenerators(t *testing.T) {
 			},
 			expectedStoresGenerator: []storeGenerator{newNodeStore, newNamespaceStore},
 		},
+		{
+			name: "Namespace from ns label as tags",
+			cfg: map[string]interface{}{
+				"kubernetes_namespace_labels_as_tags": map[string]string{
+					"env": "env",
+				},
+			},
+			expectedStoresGenerator: []storeGenerator{newNodeStore, ginewNamespaceStore},
```

Comment:
was this a typo?
```suggestion
			expectedStoresGenerator: []storeGenerator{newNodeStore, newNamespaceStore},
```

Reply:
Indeed 👍 I just removed it. Thanks for catching it